sequencer/src/api.rs

```diff
@@ -25,7 +25,9 @@ struct State<N: network::Type, Ver: StaticVersionType> {
     handle: SystemContextHandle<SeqTypes, Node<N>>,
 }
 
-impl<N: network::Type, Ver: StaticVersionType> From<&SequencerContext<N, Ver>> for State<N, Ver> {
+impl<N: network::Type, Ver: StaticVersionType + 'static> From<&SequencerContext<N, Ver>>
```

Comment:
Not a big deal, but I feel like `StaticVersionType` might as well require `'static` in the trait definition. This would need to be a change in the vbs crate so we don't need to fix it right now, but cc @nyospe